ext/standard/formatted_print.c

```diff
@@ -491,7 +491,7 @@ php_formatted_print(zval *z_format, zval *args, int argc)
 					PRINTF_DEBUG(("sprintf: getting width\n"));
 					if ((width = php_sprintf_getnumber(&format, &format_len)) < 0) {
 						efree(result);
-						php_error_docref(NULL, E_WARNING, "Width must be greater than zero and less than %d", INT_MAX);
+						zend_throw_error(NULL, "Width must be greater than zero and less than %d", INT_MAX);
```

Comment:
Same as above ``zend_value_error("Width must be greater than zero and less than %d", INT_MAX);